### PR TITLE
Implement badge dropdown

### LINKED ISSUE #68: Add UI to generate badges
Right now people don't know how to generate badges unless they've been on gitter! Make a fairly small UI for this.

### PR DESCRIPTION
**Pull request by minrk**
- add badge.svg from mybinder.org
- serve badge.svg at original `assets/images/badge.svg` and `/badge.svg` so that existing badges will not break when we swap in the the beta for at mybinder.org
- use tornado static_url macro for loading static files
- load bootstrap.js for dropdown support
- Don't add superfluous `.git` when building github URLs
- copy markdown/rst from samples from previous build page
- put shareable URL in history when submitting build so that page refresh doesn't lose state, and the shareable link should show up in user's history/search

Launch button now has a caret dropdown, which looks like this when clicked (clicking 'Launch' is unchanged):

screen shot 2017-09-26 at 14 42 37

Expanding badge snippet:

screen shot 2017-09-26 at 14 42 42

Original:

screen shot 2017-09-26 at 14 53 33

closes #68 

cc @willingc @choldgraf

### CODE REVIEW

File: binderhub/repoproviders.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def __init__(self, *args, **kwargs):
             self.repo = self.repo[:len(self.repo) - 4]
 
     def get_repo_url(self):
-        return "https://github.com/{user}/{repo}.git".format(user=self.user, repo=self.repo)
+        return "https://github.com/{user}/{repo}".format(user=self.user, repo=self.repo)
```
Comment (willingc) [inline binderhub/repoproviders.py]: Good idea.

File: binderhub/templates/index.html [hunk 2]
```diff
@@ -5,11 +5,11 @@
 <script>
  {% if submit %} window.submitBuild = true; {% endif %}
 </script>
-<link href="/static/vendor/xterm.js-2.6.0/xterm.css" rel="stylesheet" />
+<link href="{{static_url("vendor/xterm.js-2.6.0/xterm.css")}}" rel="stylesheet" />
 <!-- FIXME: vendor this -->
-<script src="/static/vendor/xterm.js-2.6.0/xterm.js"></script>
-<script src="/static/vendor/xterm.js-2.6.0/addons/fit/fit.js"></script>
-<script src="/static/index.js"></script>
+<script src="{{static_url("vendor/xterm.js-2.6.0/xterm.js")}}"></script>
```
Comment (willingc) [inline binderhub/templates/index.html]: Let's remove the comment above.

Comment (minrk) [reply binderhub/templates/index.html]: 👍

**Comment by minrk** [comment]
Thanks!

**Pull request merged**